assets/src/components/ShareButtons/ShareButtons.js

```diff
@@ -17,57 +17,61 @@ export const ShareButtons = ({social_params, social_accounts}) => {
       event: 'uaevent',
       eventCategory: 'Social Share',
       eventAction: action,
-      eventLabel: label
-    })
-  }
+      eventLabel: label,
+    });
+  };
 
   return (
     <div className="share-buttons">
       <a href={ `https://wa.me/?text=${encodeURIComponent(link)}&${utm('whatsapp', utm_medium, utm_content, utm_campaign)}` }
-        onClick={() => {share('Whatsapp', link)}}
+        onClick={() => share('Whatsapp', link)}
         target="_blank"
         className="share-btn whatsapp"
+        rel="noreferrer"
       >
-        <SvgIcon {...{name: "whatsapp"}} />
+        <SvgIcon {...{name: 'whatsapp'}} />
         <span className="visually-hidden">{__( 'Share on', 'planet4-blocks' )} Whatsapp</span>
       </a>
 
       <a href={ `https://www.facebook.com/sharer/sharer.php?u=${encodeURIComponent(link)}&${utm('facebook', utm_medium, utm_content, utm_campaign)}` }
-        onClick={() => {share('Facebook', link)}}
+        onClick={() => {share('Facebook', link);}}
```

Comment:
You can make the same change here as you did for [line 27](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1002/files#diff-2e64eedbd1a146969a412a57337e18b1dd1d738e5fc673387bbeabefbf9299a9R27), also for the 2 following `onClick` functions in the file

Reply:
Fixed

Reply:
I think that you didn't update the following `onClick` functions? There are 2 more in the file